option optimize_for = LITE_RUNTIME;

message CSPointsRacePrepareFightRequest
{
 required uint32 index = 1;
}
